src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -281,6 +287,37 @@ public static int secondSmallestMinusSmallest(final int[] values, final int defa
     }
 
     /**
+     * Returns the smallest power of 2 that exceeds or equals {@code val}
+     * @param val input value
+     * @return power of 2 integer
+     * @throws IllegalArgumentException if an int overlow is encountered
+     */
+    public static int smallestPowerOfTwoGreaterThan(final int val)
+            throws IllegalArgumentException {
+        if (val > Integer.MAX_VALUE/2 || val < 0)
+            throw new IllegalArgumentException("The mallest power of 2 greater than " + val + " is greater than Integer.MAX_VALUE" +
+                    " or negative input.");
+        return val > 1 ? Integer.highestOneBit(val - 1) << 1 : 1;
+    }
+
+    /**
+     * Shrink or expand a double array uniformly using nearest neighbor interpolation
```

Comment:
I think the description of the method could be expanded upon (no pun intended) or made more exact.